pyproject.toml

```diff
@@ -1,3 +1,111 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = ["hatchling"]
+
+[project]
+name = "basket-client"
+dynamic = ["version"]
+description = "A Python client for Mozilla's basket service."
+readme = "README.rst"
+requires-python = ">=3.7"
+license = {text = "BSD-3-Clause"}
+keywords = ["mozilla", "basket"]
+authors = [
+  { name = "Michael Kelly" },
+]
+maintainers = [
+  { name = "Paul McLanahan" },
+  { name = "Rob Hudson" },
+]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Web Environment",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: BSD License",
+  "Natural Language :: English",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Topic :: Communications",
+  "Topic :: Software Development :: Libraries",
+]
+dependencies = [
+  "requests",
+]
+
+[project.urls]
+Documentation = "https://github.com/mozilla/basket-client#readme"
+Issues = "https://github.com/mozilla/basket-client/issues"
+Source = "https://github.com/mozilla/basket-client"
+
+[tool.hatch.version]
+path = "src/basket/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
+  "src/basket/*",
+  "LICENSE",
+  "README.rst",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/basket"]
+
+[tool.hatch.envs.test]
+dependencies = [
+  "coverage[toml]>=6.5",
+  "pytest",
+  "pytest-cov",
+  "tox",
```

Comment:
I usually version my dev dependencies. It's a chore to keep up, but they do change in weird ways sometimes and debugging issues when you're using different versions of things is so frustrating.

But I know people who don't version their dev dependencies and they're probably happier people in general, so who knows.

Reply:
I'll try the happy route and see how it goes.